sw/device/silicon_creator/lib/manifest.h

```diff
@@ -400,6 +400,15 @@ enum {
   kManifestExtIdSpxKey = 0x94ac01ec,
   kManifestExtIdSpxSignature = 0xad77f84a,
   kManifestExtIdSecVerWrite = 0x3f086a41,
+
+  /**
+   * ASCII "ISFB".
+   */
+  kManifestExtIdIsfb = 0x42465349,
```

Comment:
Can you spell out what ISFB means (i.e. Integration Specified Firmware Binding) in this comment please  (same below)

Reply:
Done.